docs/man_pages/bodhi-push.rst

```diff
@@ -64,3 +64,9 @@ If you find bugs in bodhi (or in the mage page), please feel free to file a bug
 request::
 
     https://github.com/fedora-infra/bodhi
+
+
+Help
```

Comment:
This is OK as-is if you prefer, but I had been suggesting to rename the "Bugs" section above to "Help" so that we don't have so many sections with only one sentence at the end of the page.